frontend/src/components/Shared/Map/style/style.json

```diff
@@ -99,20 +99,6 @@
             "type": "line",
             "source": "openmaptiles",
             "source-layer": "waterway",
-            "filter": [
-                "all",
-                [
-                    "==",
-                    "$type",
-                    "LineString"
-                ],
-                [
-                    "in",
-                    "class",
-                    "canal",
-                    "river"
-                ]
-            ],
```

Comment:
Just want to make sure this is the only layer we have with the waterway source layer, I'm not sure if we have an incorrectly configured "canals are a different color" type layer in there.

Reply:
Yeah there's no filtering in the water stuff: https://github.com/cityofaustin/ctxfloods/blob/44910295564a39f10b6519a7a32bf70dedd8f478/frontend/src/components/Shared/Map/style/style.json